docusaurus.config.js

```diff
@@ -164,6 +162,12 @@ const config = {
         ],
         copyright: `Copyright © ${new Date().getFullYear()} wasmCloud LLC. All rights reserved. The Linux Foundation has registered trademarks and uses trademarks. For a list of trademarks of The Linux Foundation, please see our Trademark Usage page: https://www.linuxfoundation.org/trademark-usage. Built with Docusaurus.`,
       },
+      algolia: {
+        apiKey: 'f0ef30f3d98ce5e9a7dd7579bb221dfc',
```

Comment:
should this be a secret?

Reply:
yes 😞

```suggestion
        apiKey: process.env.ALGOLIA_API_KEY,
```

I believe the `appId` and `indexName` are not secured but the apiKey is used to update the index during build

Reply:
re: https://docusaurus.io/docs/search#using-algolia-docsearch it's okay
```
      // The application ID provided by Algolia
      appId: 'YOUR_APP_ID',

      // Public API key: it is safe to commit it
      apiKey: 'YOUR_SEARCH_API_KEY',
```

Reply:
![image](https://github.com/wasmCloud/wasmcloud.com-dev/assets/12040685/cc3112b2-16e3-4950-9a9a-34df9a12a9e6)
...

In our acceptance email they recommended copy/pasting this directly into the frontend as a script, maybe I misunderstand? I can regenerate that key and invalidate it if necessary :(

Reply:
As long as I didn't pick the wrong key (which pretty sure this is a search key) this is what Docusaurus does https://github.com/facebook/docusaurus/blob/main/website/docusaurus.config.js#L493

Reply:
Does algolia index the site from their side? how do the content changes get indexed on build?

Reply:
They don't get indexed on build, it actually gets indexed on a daily interval

Reply:
ok, cool, this still looks good then!